protocol/daemons/slinky/client/constants.go

```diff
@@ -0,0 +1,20 @@
+package client
+
+import "time"
+
+var (
+	// SlinkyPriceServerConnectionTimeout controls the timeout of establishing a
+	// grpc connection to the pricefeed server.
+	SlinkyPriceServerConnectionTimeout = time.Second * 5
+	// SlinkyPriceFetchDelay controls the frequency at which we pull prices from slinky and push
+	// them to the pricefeed server.
+	SlinkyPriceFetchDelay = time.Second * 2
+	// SlinkyMarketParamFetchDelay is the frequency at which we query the x/price module to refresh mappings from
+	// currency pair to x/price ID.
+	SlinkyMarketParamFetchDelay = time.Millisecond * 1900
+)
```

Comment:
curious how we arrived at above values for `SlinkyPriceFetchDelay` and `SlinkyMarketParamFetchDelay`? Have we experimented with how low these values can be?

Reply:
These are just initial values--I haven't experimented too much with them.
Here's my thoughts on how to think about them:

`SlinkyPriceFetchDelay` can be as low as we want it to be. My initial thought was that it doesn't make sense to set it lower than, however, if we use the `min_exchanges` param as the number of historical slinky prices to keep in the indexPriceCache, maybe it makes sense to hit the sidecar a bunch every block and take the median of those values via x/prices GetValidMarketPriceUpdates method.

For the `MarketParamFetchDelay` I just wanted it to refresh for frequently than we retrieved prices so there would never be a mismatch between the keeper state and the prices returned.